blender/arm/material/cycles.py

```diff
@@ -130,7 +130,8 @@ def parse_material_output(node: bpy.types.Node, custom_particle_node: bpy.types.
                     wrd.world_defs += '_EmissionShaded'
 
         if parse_opacity:
-            curshader.write('opacity = {0} - 0.0002;'.format(out_opacity))
+            curshader.write('opacity = {0};'.format(out_opacity))
```

Comment:
What's the reason for this change? Looks like the - 0.0002 fixed https://github.com/armory3d/armory/issues/1413 (https://github.com/armory3d/armory/commit/35274c86734c04a62c6cfa4836b00d9412c84f09) although it is probably more of a workaround. There is a known issue with the - 0.0002 (see https://discord.com/channels/486771218599510021/1047945916411297872) that still needs to be resolved, but this change doesn't seem related to refraction or is it?

Reply:
These kind of setup are called voodoo variable, they are called like so because they pop out at some point where no one but the author can understand it.
I have a correct transparency with my setup, it should say so.